plugins/aws/bedrock/privateCustomModel.js

```diff
@@ -0,0 +1,110 @@
+                return rcb();
+            }
+
+            if (!listCustomModels.data.length) {
+                helpers.addResult(results, 0, 'No Bedrock custom model found', region);
+                return rcb();
+            }
+            var subnetRouteTableMap;
+            var privateSubnets = [];
+
+            var describeSubnets = helpers.addSource(cache, source,
+                ['ec2', 'describeSubnets', region]);
+            var describeRouteTables = helpers.addSource(cache, {},
+                ['ec2', 'describeRouteTables', region]);
+                
+            if (!describeRouteTables || describeRouteTables.err || !describeRouteTables.data ) {
+                helpers.addResult(results, 3,
+                    'Unable to query for route tables: ' + helpers.addError(describeRouteTables), region);
+                return rcb();
+            }     
+            
+            if (!describeSubnets || describeSubnets.err || !describeSubnets.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for subnets: ' + helpers.addError(describeSubnets), region);
+                return rcb();                  
+            } else {
+                subnetRouteTableMap = helpers.getSubnetRTMap(describeSubnets.data, describeRouteTables.data);
+                privateSubnets = helpers.getPrivateSubnets(subnetRouteTableMap, describeSubnets.data, describeRouteTables.data);   
+            }
+
+            for (let model of listCustomModels.data){
+                if (!model.modelArn|| !model.modelName) continue;
+               
+                let resource = model.modelArn;
+
+                let getCustomModel = helpers.addSource(cache, source,
+                    ['bedrock', 'getCustomModel', region, model.modelName]);
+
+    
+                if (!getCustomModel || getCustomModel.err || !getCustomModel.data) {
+                    helpers.addResult(results, 3, `Unable to describe Bedrock custom model : ${helpers.addError(getCustomModel)}`, region, resource);
+                    continue;
+                }
+                
+                let getModelJob = helpers.addSource(cache, source,
+                    ['bedrock', 'getModelCustomizationJob', region, getCustomModel.data.jobArn]);
+                
+                if (!getModelJob || getModelJob.err || !getModelJob.data) {
+                    helpers.addResult(results, 3, `Unable to describe Bedrock model customzation job : ${helpers.addError(getModelJob)}`, region, resource);
+                    continue;
+                }
+
+                if (getModelJob.data.vpcConfig && getModelJob.data.vpcConfig != '') {
+
+                    var allPrivate = getModelJob.data.vpcConfig.subnetIds.every(subnetId => privateSubnets.includes(subnetId));
+                    
+                    if (allPrivate) {
+                        helpers.addResult(results, 0,
+                            'Bedrock custom model is a private model',
```

Comment:
```suggestion
                            'Bedrock custom model is configured within a private VPC',
```
should match whats in the description